.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -7,33 +7,41 @@ assignees: ['harperreed']
 - OS: [e.g. iOS, Windows, Linux]
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]
 - Weather Provider: [e.g. OpenMeteo, PirateWeather]
 
 ## Additional Context
+
 Add any other context about the problem here.
 
 ## Error Logs
+
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)